src/SoundClassifier/index.js

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2019 ml5
```

Comment:
Nitpick: whenever you're making the slightest modification of a file, you'd be entitled to update the copyright header to e.g. `2019-2023` 🙂